src/main/resources/com/cloudbees/simplediskusage/QuickDiskUsagePlugin/index.jelly

```diff
@@ -0,0 +1,79 @@
+
+    Permission is hereby granted, free of charge, to any person obtaining a copy
+    of this software and associated documentation files (the "Software"), to deal
+    in the Software without restriction, including without limitation the rights
+    to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+    copies of the Software, and to permit persons to whom the Software is
+    furnished to do so, subject to the following conditions:
+
+    The above copyright notice and this permission notice shall be included in
+    all copies or substantial portions of the Software.
+
+    THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+    IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+    FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+    AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+    LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+    OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+    THE SOFTWARE.
+
+-->
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:l="/lib/layout" xmlns:s="/lib/form">
+    <l:layout title="${%Disk usage}" permission="${app.ADMINISTER}">
+        <l:breadcrumb title="${%Disk usage}" href="."/>
+        <st:include page="sidepanel.jelly" />
+        <l:main-panel>
+            <h1>${%Disk usage}</h1>
+            <st:include page="message.jelly" />
+            <l:tabBar>
+                <l:tab name="${%Jobs}" active="true" href="."/>
+                <l:tab name="${%Directories}" href="./directories"/>
+            </l:tabBar>
+            <table class="sortable pane bigtable">
+                <thead>
+                    <tr>
+                        <th class="pane-header" initialSortDir="down">${%Item name}</th>
+                        <th class="pane-header" style="text-align: right">${%Disk usage} (kB)</th>
+                        <th class="pane-header" style="text-align: right">${%Action}</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <j:forEach var="e" items="${it.jobsUsages}">
+                        <tr>
+                            <td class="pane">
+                                <a href="${rootURL}/${e.url}" class="model-link inside">${e.displayName}</a>
+                            </td>
+                            <td class="pane" style="text-align: right">
+                                ${e.usage}
+                            </td>
+                            <td class="pane" style="text-align: right">
+                                <s:link href="clean/?job=${e.fullName}" post="true">run cleanup</s:link>
+                            </td>
+                        </tr>
+                    </j:forEach>
+                </tbody>
+            </table>
+            <p></p>
```

Comment:
In the next `<div>`you have a `style` attribute... `margin-top: 10px` :dizzy_face: